### ISSUE TITLE
Skypack cannot server faker 6.0

### ISSUE THREAD

**Issue by jupegarnica** [faker-js/faker#635]
### Describe the bug

Skypack cannot server faker 6.0 so, cannot be used from deno

### Reproduction

https://cdn.skypack.dev/@faker-js/faker@6.0.0

```js
/*
 * [Package Error] "@faker-js/faker@v6.0.0" could not be built. 
 *
 * No build output found.
 *
 * How to fix:
 *   - If you believe this to be an error in Skypack, file an issue here: https://github.com/skypackjs/skypack-cdn/issues
 *   - If you believe this to be an issue in the package, share this URL with the package authors to help them debug & fix.
 *   - Use https://skypack.dev/ to find a web-friendly alternative to find another package.
 */

console.warn("[Package Error] \"@faker-js/faker@v6.0.0\" could not be built. ");
throw new Error("[Package Error] \"@faker-js/faker@v6.0.0\" could not be built. ");
export default null;

```

**Comment by jupegarnica** [comment]
the beta was working fine:

https://cdn.skypack.dev/@faker-js/faker@6.0.0-beta.0

**Comment by Shinigami92** [comment]
Yeah I see the error :thinking: 
I will dig into this and try find out what's different between beta.0 and release

**Comment by Shinigami92** [comment]
I will compare builds between

https://unpkg.com/@faker-js/faker@6.0.0/package.json and https://unpkg.com/@faker-js/faker@6.0.0-beta.0/package.json

**Comment by Shinigami92** [comment]
I ensured that the **cjs** version is running locally in my test folder. So this bug only affects skypack somehow :thinking: 

I added the label `s: on hold` as an intent to mark this as waiting on _upstream_. We don't have a label for that (yet) and I'm not sure if we need such a label.

**Comment by hyrious** [comment]
This might be a skypack issue which depends on the "default" field existing in the export: https://github.com/skypackjs/skypack-cdn/issues/269

As a contrast, [esm.sh](https://esm.sh/@faker-js/faker@v6.0.0) and [esm.run](https://esm.run/@faker-js/faker@v6.0.0) both work fine.

**Comment by Shinigami92** [comment]
Perfect, thank you @hyrious 
We will update the links in docs to mention this links :+1:
